### PR TITLE
Add dhstore-helga

### PR DESCRIPTION
**Pull request by ischasny**
Add a new instance `dhstore-helga` in prod. It's going to be used for writes while the existing one will serve reads only.

### CODE REVIEW

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            nodeSelectorTerms:
+              - matchExpressions:
+                  - key: node.kubernetes.io/instance-type
+                    operator: In
+                    values:
+                      - r5n.2xlarge
+                  - key: topology.kubernetes.io/zone
+                    operator: In
+                    values:
+                      - us-east-2c
```
Comment (masih) [inline deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/deployment.yaml:49]: Checking this is different from the current AZ used by the existing dhstore?

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/pvc.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dhstore-data
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 20Ti
```
Comment (masih) [inline deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/pvc.yaml:10]: Max is 16 right?

Regardless, let's start with a small number and increase as needed.

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/deployment.yaml (hunk 1 above)
Comment (ischasny) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/deployment.yaml:49]: I think it's the same. Should it be different?

File: deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/pvc.yaml (hunk 2 above)
Comment (ischasny) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/pvc.yaml:10]: Good point - will decrease.

Comment (ischasny) [reply deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore-helga/pvc.yaml]: Done

**Review by masih — APPROVED** [review]

**Pull request merged**